README.md

```diff
@@ -1,3 +1,35 @@
-# DevOps
+# Helpful Engineering DevOps Team
 
-Repository for automation scripts.
+## FAQ
+
+**How do I get help?**
+
+Click the lightning bolt in the header of the slack channel and select “Help Request”
+
+**How do I help out?**
+
+Fill out our skill survey at https://airtable.com/shrisE8yL9W3cegHr and check out our Trello board at https://trello.com/invite/b/8TEQ0gH0/82b3cce6da639b5cebde085def1d2ede/skill-software-devops-projects
+
+**When are check-in meetings?**
+
+Daily at 1500 PT/2200 GMT on zoom: https://zoom.us/j/471016936?pwd=cXh2RWFib2F4RU1ldVZPM3BNdkcrQT09
+
+**Who should I ask about X?**
+
+Check out the directory below
+
+##Directory
+
+### Services
+
+- AWS: @Chris Mac @skuenzli
+- Azure: @Jonathan Vermeij @Troy Witthoeft
+- Central project & volunteer list/AirTable: @psprings @jonnyparris
```

Comment:
Think @ken should be added here. Worth checking  with @psprings that @jonnyparris is working on the same system